modular_skyrat/master_files/code/modules/client/preferences/erp_preferences.dm

```diff
@@ -90,9 +90,18 @@
 
 /datum/preference/toggle/erp/breast_enlargement
 	savefile_key = "breast_enlargement_pref"
+	
+/datum/preference/toggle/erp/breast_shrinkage
+	savefile_key = "breast_shrinkage_pref"
 
 /datum/preference/toggle/erp/penis_enlargement
 	savefile_key = "penis_enlargement_pref"
+	
+/datum/preference/toggle/erp/penis_shrinkage
```

Comment:
the prefs on 94 and 100 are unneeded, meh on 103

Reply:
I disagree here. The whole reason I started this was a) because bugs and b) the options were not modular enough for one of my friends. I assure you these are not superfluous, I spent a lot of time on the logic.